src/diffusers/pipelines/marigold/marigold_image_processing.py

```diff
@@ -0,0 +1,152 @@
+from typing import Optional, Tuple, Union
+
+import numpy as np
+import torch
+import torch.nn.functional as F
+from PIL import Image
+
+from ... import ConfigMixin
+from ...configuration_utils import register_to_config
+from ...utils import CONFIG_NAME, logging
+
+
+logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
+
+
+class MarigoldImageProcessor(ConfigMixin):
```

Comment:
(nit): should this go to https://github.com/huggingface/diffusers/blob/main/src/diffusers/image_processor.py?

Reply:
I took after the placement of https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/blip_diffusion/blip_image_processing.py - `MarigoldImageProcessor` also is specific to the Marigold family. Would you want me to move it into the file you referenced? No preference from my side, let me know how to go about it

Reply:
I think it is ok here because it is not inherited from our image processor; and I think it is ok to not inherit from our `VaeImageProcessor` because it is pretty different and can't re-use most of our methods